apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -256,17 +266,16 @@ init_per_group(_, Config0) ->
             (StartHeight - ParentTopHeight) + ?PARENT_FINALITY),
     [ {staker_names, [?ALICE, ?BOB, ?LISA]}, {parent_start_height, StartHeight} | Config].
 
-child_node_config(Node, Stakeholders, CTConfig) ->
+child_node_config(Node, Stakeholders, Pinners, CTConfig) ->
     ReceiveAddress = encoded_pubkey(?FORD),
-    Pinning = false,
-    NodeConfig = node_config(Node, CTConfig, Stakeholders, ReceiveAddress, Pinning),
+    NodeConfig = node_config(Node, CTConfig, Stakeholders, Pinners, ReceiveAddress),
     build_json_files(?HC_CONTRACT, NodeConfig, CTConfig),
     aecore_suite_utils:create_config(Node, CTConfig, NodeConfig, [{add_peers, true}]).
 
 end_per_group(_Group, Config) ->
     Config1 = with_saved_keys([nodes], Config),
     [ aecore_suite_utils:stop_node(Node, Config1)
-      || {Node, _, _} <- proplists:get_value(nodes, Config1, []) ],
```

Comment:
Is the fourth value here used anywhere?

Reply:
Yes, there'a a fourth value used there, which is essentially the `Pinners` list you saw above. It has a fourth member in the latest commit.

Reply:
Yes, found the one place where it isn't `_` - at some point we might consider using a map/record instead of a tuple 🙈